schedule/virt_autotest/slem6_baremetal_usb_install.yaml

```diff
@@ -0,0 +1,10 @@
+name: slem6_baremetal_usb_install.yaml
+description:    >
+    Maintainer: xlai@suse.com, qe-virt@suse.de
+    SLE Micro 6.0 bare metal machine installation with Self-Install iso schedule
+schedule:
+    - installation/ipxe_install
+    - installation/usb_install
+    - installation/bootloader_uefi
+    - microos/selfinstall
+    # system is now ready: ssh password login is ok, grub2 menu can show in sol
```

Comment:
I think this comment is not need for schedule itself.

Reply:
Well, that's true. But since the consumers of this PR care a lot the status of the machine after installation, so comment it here just for easy awareness.
So I tend to keep it if you are fine.